Mathlib/Algebra/GroupWithZero/Action/Defs.lean

```diff
@@ -327,7 +327,15 @@ def DistribSMul.toAddMonoidHom (x : M) : A →+ A :=
 
 end DistribSMul
 
-/-- Typeclass for multiplicative actions on additive structures. This generalizes group modules. -/
+/-- Typeclass for multiplicative actions on additive structures.
+
+For example, if `G` is a group (with group law written as multiplication) and `A` is an
+abelian group (with group law written as addition), then the way to give `A` a `G`-module
+structure (for example, to use the theory of group cohomology) is to say `[DistribMulAction G A]`.
+
+Mathematically, `DistribMulAction G A` is equivalent to giving `A` the structure of
+a `ℤ[G]`-module.
+-/
 @[ext]
 class DistribMulAction (M A : Type*) [Monoid M] [AddMonoid A] extends MulAction M A where
```

Comment:
Do you think it's worth a comment here about how this is often used where `Module` might be expected?

Reply:
Works for me, if that's the wording a mathematician would understand.